tests/testdata/csv/cities_NL_case_flip.csv

```diff
@@ -0,0 +1,11 @@
+Name,no_Latitude,EPSG,no_Longitude,NumberOfCitizens,TimeStamp
```

Comment:
For flipping, wouldn't it make the test a bit more realistic that the coordinate names are "x" and "y" ? VERY minor issue.